sycl/test-e2e/KernelCompiler/kernel_compiler_sycl_jit_cache.cpp

```diff
@@ -76,15 +71,10 @@ int test_persistent_cache() {
   // CHECK: [kernel_compiler Persistent Cache]: cache miss: [[KEY1:.*]]
   // CHECK: [kernel_compiler Persistent Cache]: storing device code IR: {{.*}}/[[KEY1]]
   exe_kb kbExe1a = syclex::build(kbSrc1);
-  dumpKernelIDs();
```

Comment:
does it make sense to replace it with check get_kernel_ids().size() == 0?

Reply:
I've added `test_no_visible_ids` in this PR which asserts for `get_kernel_ids().size() == 0`, given that this is now always true regardless of persistent checks etc, I think we are already covered